nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -359,6 +359,9 @@
 - The `antennas` package and the `services.antennas` module have been
   removed as they only work with `tvheadend` (see above).
 
+- `/tmp` is now mounted on a `tmpfs`, which is more in-line with both the File Hierarchy Standard (FHS) and most distros.
+  - `nix-daemon` will be manually set to use `/var/tmp` on Linux, although this should ultimately be configured better by `nix` itself.
```

Comment:
I don't see this happening in this PR. Is something missing here?

Reply:
The "tmp on tmpfs" change: (https://github.com/NixOS/nixpkgs/pull/338181/commits/94d970d3e07d384515eb96ca8dd0dd79ba53c27f)

Like mentioned in my latest comment, I am not quite sure where to change Nix-daemon its default TMPDIR, but once I do I'll make that secondary change and reorder the commits. Or wait for https://github.com/NixOS/nix/issues/8995

Reply:
I think [around the `CA_BUNDLE` env var override](https://github.com/NixOS/nixpkgs/blob/02e5f3cda511d5096a1d0334396591660efb2361/nixos/modules/services/system/nix-daemon.nix#L190) would do nicely since [the `nix.envVars` option seems to be applied on a wider scale](https://github.com/NixOS/nixpkgs/blob/02e5f3cda511d5096a1d0334396591660efb2361/nixos/modules/services/system/nix-daemon.nix#L242) which would affect much more than just the nix-daemon.

Adding a separate option for this probably isn't necessary since one can `lib.mkForce` that specific environment variable<sup>[citation needed]</sup>.